ci/azure-pipelines.yml

```diff
@@ -212,6 +212,30 @@ jobs:
   - publish: ./ci/projects/kmac_sca_random_cw310_ujson.html
     artifact: traces_kmac_random_cw310_ujson
     displayName: "Upload KMAC Random traces (uJSON)"
+- job: ceca
+  displayName: "CECA Attack"
+  timeoutInMinutes: 30
+  pool:
+    name: FPGA SCA
+    demands: BOARD -equals cw310
+  steps:
+  - checkout: self
+  - bash: |
+      python3 -m pip install --user -r python-requirements.txt
+    displayName: "Install python dependencies"
+  - bash: |
+      apt update
+      apt install git-lfs
+    displayName: "Install system dependencies"
+  - bash: |
+      git-lfs pull
+    displayName: "Pull LFS binaries"
+  - bash: |
+      set -e
+      pushd ci
+      ../analysis/ceca.py -f ci_data/ceca/aes_cw310.db -d output -s 3 -w 4 -a 1060 1119 -n 16000
```

Comment:
Have you tried this with the CW305 or CW310? There should be a huge difference between the two. BTW, all traces I provided to you were taken from the CW305.

Reply:
Oh nicely spotted. Indeed, this trace dataset is from CW305 not CW310 - I'll fix that.